generators/entity-client/templates/angular/src/main/webapp/app/entities/pagination-template.ejs

```diff
@@ -56,8 +56,8 @@
             this.predicate = data.pagingParams.predicate;
         });
         <%_ if (searchEngine === 'elasticsearch') { _%>
-        this.currentSearch = this.activatedRoute.snapshot && this.activatedRoute.snapshot.params['search'] ?
-            this.activatedRoute.snapshot.params['search'] : '';
+        this.currentSearch = this.activatedRoute.snapshot && this.activatedRoute.snapshot.queryParams['search'] ?
```

Comment:
can you pls apply similar fix in the `entities/infinite-scroll-template.ejs` and `entities/no-pagination-template.ejs`

Reply:
OK I'll do it for you.